web/app/views.py

```diff
@@ -369,47 +363,57 @@ def upload_data(return_page: str):
     """
     warnings, errors = [], []
 
-    data = request.files['data']
+    current_user = request.cookies.get('currentUser')
+
+    if not current_user:
+        # Use DEFAULT as current user if the client's cookie is not set.
+        current_user = 1
+
+    dataset = request.files['dataset']
 
     with NamedTemporaryFile() as temp_file:
-        data.save(temp_file.name)
-        data_errors = validate_data(temp_file.name)
+        dataset.save(temp_file.name)
+        dataset_errors = validate_data(temp_file.name)
 
-        if len(data_errors) > 0:
-            errors.extend(data_errors)
+        if len(dataset_errors) > 0:
+            errors.extend(dataset_errors)
         else:
-            data_name = secure_filename(data.filename)
-            original_data_path = os.path.join(app.config['DATA_FOLDER'], data_name)
+            dataset_name = request.form['datasetName']
+            # dataset_class = load_args(ckpt).dataset_type #TODO: SWITCH TO ACTUALLY FINDING THE CLASS
```

Comment:
By now I'm sure you're incredibly annoyed by all my nitpicky comments, so here's another one for you. The PEP standard says comments at the end of a line should have two spaces before the hashtag and one after. Again I don't care too much, but my IDE will complain.
```suggestion
            # dataset_class = load_args(ckpt).dataset_type  # TODO: SWITCH TO ACTUALLY FINDING THE CLASS
```